Sources/StreamChat/WebSocketClient/Events/NotificationEvents.swift

```diff
@@ -70,76 +70,20 @@ public struct NotificationAddedToChannelEvent: EventWithChannelId, EventWithPayl
     }
 }
 
-public struct NotificationRemovedFromChannelEvent: EventWithChannelId {
+public struct NotificationRemovedFromChannelEvent: UserEvent, ChannelEvent {
+    public let userId: UserId
```

Comment:
user id for notificaiton events is always currrent user ID AFAIK (please check with Toly)